### PR TITLE
Update auth regarding method isRouteNotSecure

### PR DESCRIPTION
**Pull request by szymonbrainhub**
Change in documentation of `auth` module regarding this PR: https://github.com/brainhubeu/hadron/pull/44/files

- [x] Added note about depreciation
- [x] Added information about new method

### CODE REVIEW

File: docs/packages/auth.md [hunk 1]
```diff
@@ -567,31 +567,32 @@ If, on the other hand, you want the authorization to succeed just return `next()
...
 To simplify working with paths and roles, hadron-auth exposes two utility functions (that can be `require`d from the hadron-auth module).
 
-- `isRouteNotSecure(path)` checks if `path` even exists in the `securedRotes` key.
+- `isRouteSecure(path)` checks if `path` exists in the `securedRotes` key.
 - `isAllowed(path, method, user, allRoles)` checks if the user is allowed access to the route where:
   - `path` is the route path (usually `req.path`)
   - `method` is the route method (usually `req.method`)
   - `user` is the user object that has the `roles` key (see [Schemas](#schemas))
   - `allRoles` is an array of names of all roles in the database
+- ~~`isRouteNotSecure(path)`~~ (deprecated) checks if `path` even exists in the `securedRotes` key.
```
Comment (marekbrainhub) [inline docs/packages/auth.md]: I'm uncertain about the wording on this one, doesn't seem descriptive.

Comment (szymonbrainhub) [reply docs/packages/auth.md]: Maybe I'll remove the description at all. Let's leave just deprecation note. What do You think?

Comment (piotr-s-brainhub) [reply docs/packages/auth.md]: rather `checks if `path` doesn't exist`

Comment (szymonbrainhub) [reply docs/packages/auth.md]: Yeah, that sound much better. I'll fix that

**Pull request merged**